cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -134,7 +133,7 @@ public AppFabricHttpHandler(Authenticator authenticator, CConfiguration configur
     this.queueAdmin = queueAdmin;
     this.txClient = txClient;
     this.dsFramework =
-      new NamespacedDatasetFramework(dsFramework, new DefaultDatasetNamespace(configuration, Namespace.USER));
+      new NamespacedDatasetFramework(dsFramework, new DefaultDatasetNamespace(configuration));
```

Comment:
and if we use both, maybe it makes sense to pass in NamespacedDatasetFramework directly when needed instead of DatasetFramework.